CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - You can skip `--name` flag when using `account import` - a default name will be generated.
 - Addresses outputted when calling `sncast account create`, `sncast deploy` and `sncast declare` are now padded to 64 characters length and prefixed with `0x0`
+- Globally available configuration to store profiles to share between projects.
+- Missing fields in `show-config` command, `block_explorer` and `show_explorer_links`.
+- Default free sepolia testnet RPC URL for any `sncast` command. 
```

Comment:
Even if we decide to add it, which I am strongly against, what is the point of mentioning this in the changelog?

Reply:
I suppose that it is a significant change. It changes the expected behavior from the older version. Isn't it proper to mention here?

Reply:
I see now that this also applies to the default `CastConfig`. I thought it was only for the template. That makes it even worse, but in such a case it should be added to the changelog.